app/src/main/res/layout/payment_methods_layout.xml

```diff
@@ -216,41 +214,29 @@
         </androidx.constraintlayout.widget.ConstraintLayout>
 
         <View
-            android:id="@+id/bonus_background"
+            android:id="@+id/view_bottom_separator"
             android:layout_width="match_parent"
-            android:layout_height="0dp"
-            android:background="@color/styleguide_payments_background"
-            android:clickable="true"
-            android:focusable="true"
-            app:layout_constraintBottom_toTopOf="@id/bottom_separator_buttons"
-            app:layout_constraintTop_toTopOf="@id/bonus_layout"
+            android:layout_height="6dp"
+            android:background="@drawable/background_shadow_payment"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/content_main"
```

Comment:
Here it can be constrained to the bottom, so that it overlaps ;) 

app:layout_constraintBottom_toBottomOf="@id/content_main"